crates/core/src/db/datastore/locking_tx_datastore/datastore.rs

```diff
@@ -1231,23 +1198,59 @@ mod tests {
         ]
     }
 
-    fn basic_table_schema() -> RawTableDefV8 {
-        RawTableDefV8::new("Foo".into(), map_array(basic_table_schema_cols()))
-            .with_indexes(vec![
-                RawIndexDefV8 {
-                    columns: ColList::new(0.into()),
+    fn basic_table_schema() -> TableSchema {
+        TableSchema::new(
+            TableId::SENTINEL,
+            "Foo".into(),
+            map_array(basic_table_schema_cols()),
+            vec![
+                IndexSchema {
+                    index_id: IndexId::SENTINEL,
+                    table_id: TableId::SENTINEL,
                     index_name: "id_idx".into(),
-                    is_unique: true,
-                    index_type: IndexType::BTree,
+                    index_algorithm: IndexAlgorithm::BTree(BTreeAlgorithm { columns: col_list![0] }),
                 },
-                RawIndexDefV8 {
-                    columns: ColList::new(1.into()),
+                IndexSchema {
+                    index_id: IndexId::SENTINEL,
+                    table_id: TableId::SENTINEL,
                     index_name: "name_idx".into(),
-                    is_unique: true,
-                    index_type: IndexType::BTree,
+                    index_algorithm: IndexAlgorithm::BTree(BTreeAlgorithm { columns: col_list![1] }),
+                },
+            ],
+            vec![
+                ConstraintSchema {
+                    table_id: TableId::SENTINEL,
+                    constraint_id: ConstraintId::SENTINEL,
+                    constraint_name: "id_constraint".into(),
+                    data: ConstraintData::Unique(UniqueConstraintData {
+                        columns: col_list![0].into(),
+                    }),
                 },
-            ])
-            .with_column_sequence(ColId(0))
+                ConstraintSchema {
+                    table_id: TableId::SENTINEL,
+                    constraint_id: ConstraintId::SENTINEL,
+                    constraint_name: "name_constraint".into(),
+                    data: ConstraintData::Unique(UniqueConstraintData {
+                        columns: col_list![1].into(),
+                    }),
+                },
+            ],
+            vec![SequenceSchema {
+                sequence_id: SequenceId::SENTINEL,
+                table_id: TableId::SENTINEL,
+                col_pos: 0.into(),
+                sequence_name: "id_sequence".into(),
+                start: 1,
+                increment: 1,
+                min_value: 1,
+                max_value: 170141183460469231731687303715884105727,
```

Comment:
NIT: Replace with `i128::MAX`

Reply:
I was just copying from the other cases.